Doc/library/xml.etree.elementtree.rst

```diff
@@ -24,6 +24,10 @@ for parsing and creating XML data.
    maliciously constructed data.  If you need to parse untrusted or
    unauthenticated data see :ref:`xml-vulnerabilities`.
 
+.. warning::
+
+   The :mod:`xml.etree.ElementTree` will be removed from python 3.9.
```

Comment:
It's `cElementTree` that was planned to be removed from Python 3.9 . But it was reverted from Python 3.9. Please see the discussion at https://bugs.python.org/issue36543